suzieq/engines/pandas/ospf.py

```diff
@@ -280,41 +290,42 @@ def aver(self, **kwargs):
                     if (x["hostname_y"] is not np.nan and
                         len(x['hostname_y']) != 1) else [], axis=1))
 
-        # Now  peering match
-        lldp_df = self._get_table_sqobj('lldp').get(
-            namespace=kwargs.get('namespace', []), columns=['default'])
+        passed_df = ospf_df \
+            .query('adjState.isin(["full", "passive"])') \
+            .reset_index(drop=True)
+        failed_df = ospf_df \
+            .query('~adjState.isin(["full", "passive"]) and '
+                   'ifState != "down"') \
+            .reset_index(drop=True)
+        ifdown_df = ospf_df \
+            .query('~adjState.isin(["full", "passive"]) and '
+                   'ifState == "down"') \
+            .reset_index(drop=True)
+        ifdown_df['assertReason'] += \
+            ifdown_df.ifState.apply(lambda x: ['ifdown'])
+
+        # Validate loopback interfaces are of the right network type
+        passed_df['assertReason'] += passed_df.apply(
+            lambda x: ['network type !loopback']
+            if (any(ele in x.ifname for ele in ['lo', 'oopbac']) and
```

Comment:
is it right the array `['lo', 'oopbac']` or it is a typo on the second element?

Reply:
Yes, some NOS have lo as their loopback name, others use Loopback, and some other loopback